ztp/kube-compare-reference/Makefile

```diff
@@ -24,4 +27,32 @@ metadata_lint: kubectl-cluster_compare
 
 .PHONY: clean
 clean:
-	rm -rf kubectl-cluster_compare
+	rm -rf kubectl-cluster_compare Chartv1 renderedv1
+
+
+.PHONY: convert
+convert: $(HELMCONVERTOR) helm
+	@if [ -e default_value.yaml ]; then \
+		$(HELMCONVERTOR) -r ./metadata.yaml -n Chartv1 -v default_value.yaml; \
+	else \
+		echo "Error: default_value.yaml doesn't exist!"; \
+		exit 0; \
```

Comment:
If it's an error, we should not exit 0, we should exit 1.  It may be possible to just call:

```
    $(HELMCONVERTOR) -r ./metadata.yaml -n Chartv1 -v default_value.yaml
```

And if it fails because there's no default_value.yaml that's okay.